components/src/dynamo/sglang/tests/test_sglang_unit.py

```
@@ -1424,3 +1426,106 @@ class _Host(LoraMixin):
+def test_set_forward_pass_metrics_worker_id_on_resolved_server_args():
+    """The publisher path end to end against a frozen ``ServerArgs``."""
+    server_args = _ResolvedServerArgs(
+        enable_forward_pass_metrics=True,
+        forward_pass_metrics_worker_id=None,
+        forward_pass_metrics_ipc_name=None,
+    )
+    endpoint = SimpleNamespace(connection_id=lambda: 4242)
+
+    set_forward_pass_metrics_worker_id(server_args, endpoint)
+
+    assert server_args.forward_pass_metrics_worker_id == "4242"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "forward_pass_metrics_worker_id"

(S105)